modules/oe_content_call_proposals/README.md

```diff
@@ -0,0 +1,51 @@
+# OpenEuropa Call for proposals Content
+
+This module provides the Call for proposals content type.
+
+## Installation
+
+Before enabling this module, make sure the following modules are present in your codebase by adding them to your
+`composer.json` and by running `composer update`:
+
+```json
+"require": {
+    "drupal/entity_reference_revisions": "~1.3",
+    "drupal/composite_reference": "~1.0-alpha2",
+    "drupal/inline_entity_form": "~1.0-rc7",
+    "drupal/field_group": "~3.0",
+}
+```
+
+The `inline_entity_form` module requires the following patches to be applied:
+
+```json
+"patches": {
```

Comment:
This patches are not required anymore, they were merged and the dependency got updated. Check the composer json and update the doc accordingly.

Reply:
I removed the patch.